### PR TITLE
Feature/tic tac game

### PR DESCRIPTION
**Pull request by TaisiaKishynskaya**

### CODE REVIEW

**Comment by TaisiaKishynskaya** [comment]
Checks crashed, b I can`t resolve this issue.
@TaisiaKishynskaya 
you need to add kivy instalation to your github actions config.

File: toe.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from kivy.lang import Builder
+from kivymd.app import MDApp
+
+
+class MainApp(MDApp):
+    TITLE = 'Tic Tac Toe!'
+    BUTTON_IDS = ['btn1', 'btn2', 'btn3', 'btn4', 'btn5', 'btn6', 'btn7', 'btn8', 'btn9']
```
Comment (vloooo) [inline toe.py:7]: почти хорошо!

File: toe.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        # Keep track of winners or losers
+        wins = {'X': 0, 'O': 0}
+        if val_a.text in wins:
+            wins[val_a.text] += 1
+        self.root.ids.game.text = f'X Wins: {wins["X"]} | O Wins: {wins["O"]}'
+
+    def disable_all_buttons(self):
+        # Disable the buttons
+        for button_id in self.BUTTON_IDS:
+            self.root.ids[button_id].disabled = True
```
Comment (vloooo) [inline toe.py]:
```suggestion
    for b_id in range(1, 10):
        self.root.ids[f'btn{b_id}'].disabled = True
```

File: toe.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            btn.text = 'O'
+            btn.disabled = True
+            self.root.ids.score.text = "X's Turn!"
+            self.turn = 'X'
+        self.win()  # Check to see if won
+
+    def restart(self):
+        self.turn = 'X'  # Reset Who`s turn it is
+
+        for button_id in self.BUTTON_IDS:
```
Comment (vloooo) [inline toe.py]: use range

File: toe.py (hunk 1 above)
Comment (vloooo) [reply toe.py:7]: use range instead of this constant

**Pull request closed without merging**